src/include/com/amazonaws/kinesis/video/webrtcclient/Stats.h

```diff
@@ -0,0 +1,322 @@
+                               //!< that are sent in order to verify consent. The average round trip time can be computed from
+                               //!< totalRoundTripTime by dividing it by responsesReceived.
+    DOUBLE currentRoundTripTime; //!< Latest round trip time (seconds)
+    DOUBLE availableOutgoingBitrate; //!< Total available bit rate for all the outgoing RTP streams on this candidate pair. Calculated by underlying
+                                     //!< congestion control
+    DOUBLE availableIncomingBitrate; //!< Total available bit rate for all the outgoing RTP streams on this candidate pair. Calculated by underlying
+                                     //!< congestion control
+    UINT64 requestsReceived; //!< Total number of connectivity check requests received (including retransmission)
+    UINT64 requestsSent; //!< The total number of connectivity check requests sent (without retransmissions).
+    UINT64 responsesReceived; //!< The total number of connectivity check responses received.
+    UINT64 responsesSent; //!< The total number of connectivity check responses sent.
+    UINT64 retransmissionsReceived; //!< The total number of connectivity check request retransmissions received
+    UINT64 retransmissionsSent; //!< The total number of connectivity check request retransmissions sent.
+    UINT64 consentRequestsSent; //!< The total number of consent requests sent.
+    UINT64 consentExpiredTimestamp; //!< The timestamp at which the latest valid STUN binding response expired
+    UINT64 bytesDiscardedOnSend; //!< Total number of bytes for this candidate pair discarded due to socket errors
+} RtcIceCandidatePairStats, *PRtcIceCandidatePairStats;
+
+/**
+ * @brief: RtcIceServerStats Stats related to the ICE Server
+ *
+ * Reference: https://www.w3.org/TR/webrtc-stats/#ice-server-dict*
+ */
+typedef struct {
+    CHAR url[MAX_STRING_LENGTH + 1]; //!< STUN/TURN server URL
+    CHAR protocol[MAX_STRING_LENGTH + 1]; //!< Valid values: UDP, TCP
+    INT32 port; //!< Port number used by client
+    UINT64 totalRequestsSent; //!< Total amount of requests that have been sent to the server
+    UINT64 totalResponsesReceived; //!< Total number of responses received from the server
+    UINT64 totalRoundTripTime; //!< Sum of RTTs of all the requests for which response has been received
+} RtcIceServerStats, *PRtcIceServerStats;
+
+/**
+ * @brief: RtcIceCandidateStats Stats related to a specific candidate in a pair
+ *
+ * Reference: https://www.w3.org/TR/webrtc-stats/#icecandidate-dict*
+ */
+
+typedef struct {
+    CHAR url[MAX_STRING_LENGTH + 1]; //!< For local candidates this is the URL of the ICE server from which the candidate was obtained
+    CHAR transportId[MAX_STRING_LENGTH + 1]; //!< ID of object that was inspected for RTCTransportStats
+    BYTE address[MAX_ADDRESS_LENGTH + 1]; //!< IPv4 or IPv6 address of the candidate
```

Comment:
this doesn't need to have + 1 size

Reply:
Just adding it explicitly to denote the NULL character.

Reply:
I don't think this is NULL terminated - this is the byte array of the address.